vulture/core.py

```diff
@@ -425,6 +448,15 @@ def is_identifier(s):
                 if is_identifier(attr):
                     self.used_attrs.add(attr)
 
+
+    def visit_While(self, node):
+        if condition_is_unsatisfiable(node.test):
+            self._define(self.unreachable_code, 'while', node.lineno,
+                size=lines.count_lines(node),
+                message="unsatisfiable 'while' condition",
+                confidence=DEFAULT_CONFIDENCE)
```

Comment:
It is unable to calculate conditions based on dynamic variables, for ex.: `if __name__ == '__main__'`, I am not sure, but there would be a similar case with env variables.

Can we use 90% instead?